examples/S2A_MSIL2A_20251008T100041_N0511_R122_T32TQM_20251008T122613_single_asset.json

```diff
@@ -0,0 +1,383 @@
+    "https://stac-extensions.github.io/view/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/processing/v1.2.0/schema.json",
+    "https://stac-extensions.github.io/product/v0.1.0/schema.json",
+    "https://stac-extensions.github.io/scientific/v1.0.0/schema.json",
+    "https://cs-si.github.io/eopf-stac-extension/v1.2.0/schema.json",
+    "https://stac-extensions.github.io/raster/v1.1.0/schema.json"
+  ],
+  "id": "S2A_MSIL2A_20251008T100041_N0511_R122_T32TQM_20251008T122613",
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [11.430710453033498, 42.426911995973605],
+        [12.763136432704886, 42.39088066238852],
+        [12.705562298541299, 41.40403474612011],
+        [11.393460776835221, 41.43884624479472],
+        [11.430710453033498, 42.426911995973605]
+      ]
+    ]
+  },
+  "bbox": [
+    11.393460776835221, 41.40403474612011, 12.763136432704886,
+    42.426911995973605
+  ],
+  "properties": {
+    "gsd": 10.0,
+    "created": "2025-10-08T12:26:13Z",
+    "mission": "copernicus",
+    "sci:doi": "10.5270/S2_-znk9xsj",
+    "updated": "2025-10-08T12:26:13Z",
+    "datetime": "2025-10-08T10:00:41.024000Z",
+    "platform": "sentinel-2a",
+    "grid:code": "MGRS-32TQM",
+    "proj:bbox": [
+      11.393460776835221, 41.40403474612011, 12.763136432704886,
+      42.426911995973605
+    ],
+    "proj:code": "EPSG:32632",
+    "providers": [
+      {
+        "url": "https://commission.europa.eu/",
+        "name": "European Commission",
+        "roles": ["licensor"]
+      },
+      {
+        "url": "https://sentinel.esa.int/web/sentinel/missions/copernicus",
+        "name": "ESA",
+        "roles": ["producer", "processor"]
+      },
+      {
+        "url": "https://zarr.eopf.copernicus.eu/",
+        "name": "EOPF Sentinel Zarr Samples Service",
+        "roles": ["host", "processor"]
+      }
+    ],
+    "published": "2025-10-08T12:26:13Z",
+    "instruments": ["msi"],
+    "end_datetime": "2025-10-08T10:00:41.024000Z",
+    "product:type": "S02MSIL2A",
+    "constellation": "sentinel-2",
+    "eo:snow_cover": 0.000305,
+    "mgrs:utm_zone": 32,
+    "proj:centroid": {
+      "lat": 41.91644,
+      "lon": 12.0732
+    },
+    "eo:cloud_cover": 0.003839,
+    "start_datetime": "2025-10-08T10:00:41.024000Z",
+    "sat:orbit_state": "descending",
+    "eopf:datatake_id": "GS2A_20251008T100041_053773_N05.11",
+    "mgrs:grid_square": "QM",
+    "processing:level": "L2A",
+    "view:sun_azimuth": 163.595949089503,
+    "mgrs:latitude_band": "T",
+    "processing:lineage": "IPF L2A processor",
+    "product:timeliness": "PT3H",
+    "sat:absolute_orbit": 53773,
+    "sat:relative_orbit": 122,
+    "view:sun_elevation": 49.3049375691464,
+    "processing:facility": "ESA",
+    "processing:software": {
+      "Sentinel-2 IPF": " "
+    },
+    "eopf:instrument_mode": "INS-NOBS",
+    "product:timeliness_category": "NRT",
+    "sat:platform_international_designator": "2015-028A"
```

Comment:
nit: it would be easier to read this list of properties if it was alphabetically sorted.